linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -296,45 +304,66 @@ impl LocalKubernetesNet {
-            )?;
-
-            HelmRelease::install(
-                String::from("linera-core"),
-                &base_dir,
-                i,
-                &github_root,
-                self.num_shards,
-                cluster_id,
-            )
-            .await?;
+        let mut validators_initialization_futures = Vec::new();
+        for (i, kind_cluster) in self.kind_clusters.iter().cloned().enumerate() {
+            let docker_image_name = docker_image.get_name().to_string();
+            let base_dir = base_dir.clone();
+            let github_root = github_root.clone();
+
+            let next_kubectl_instance = kubectl_instance.clone();
+            let next_tmp_dir_path = tmp_dir_path.clone();
+
+            let future = async move {
+                let kind_cluster = kind_cluster.lock().await;
+                let cluster_id = kind_cluster.id();
+                kind_cluster.load_docker_image(&docker_image_name).await?;
 
-            // Query the cluster
-            let output = self.kubectl_instance.get_pods(cluster_id).await?;
-            let validator_pod_name = output
-                .split_whitespace()
-                .find(|&t| t.contains("proxy"))
-                .expect("Getting validator pod name should not fail");
-
-            let local_port = 19100 + i;
-            self.kubectl_instance
-                .port_forward(
-                    validator_pod_name,
-                    &format!("{local_port}:{local_port}"),
+                let server_config_filename = format!("server_{}.json", i);
+                fs::copy(
+                    tmp_dir_path.join(&server_config_filename),
+                    base_dir.join(&server_config_filename),
+                )?;
+
+                HelmRelease::install(
+                    String::from("linera-core"),
+                    &base_dir,
+                    i,
+                    &github_root,
+                    num_shards,
                     cluster_id,
                 )
                 .await?;
+
+                let mut kubectl_instance = kubectl_instance.lock().await;
+                let output = kubectl_instance.get_pods(cluster_id).await?;
+                let validator_pod_name = output
+                    .split_whitespace()
+                    .find(|&t| t.contains("proxy"))
+                    .expect("Getting validator pod name should not fail");
+
+                let local_port = 19100 + i;
+                kubectl_instance
+                    .port_forward(
+                        validator_pod_name,
+                        &format!("{local_port}:{local_port}"),
+                        cluster_id,
+                    )
+                    .await?;
+
+                Result::<(), anyhow::Error>::Ok(())
+            };
+
+            validators_initialization_futures.push(future);
+
+            kubectl_instance = next_kubectl_instance;
+            tmp_dir_path = next_tmp_dir_path;
```

Comment:
I don't get this part

Reply:
Yeah, maybe I overcomplicated this. Let me change it.
But to explain, so we clone these outside of the for loop (so we don't use `self` inside the loop and move it, then in the next iteration it can't be used). One option that I think might be better would be to clone it again at the start of the loop, and use it. However because of naming I decided to clone it, use the one cloned in the previous iteration (or outside of the loop, if it's the first), then at the end replace it. But I see now how that's more complicated to understand.
I'll change the approach to just clone in the start and use it